jetson/percep/main.cpp

```diff
@@ -8,10 +8,27 @@ using namespace cv;
 using namespace std;
 using namespace std::chrono_literals;
 
+rapidjson::Document mRoverConfig;
+ 
 int main() {
+ // reading in configuration file
+ ifstream configFile;
+   string configPath = getenv("MROVER_CONFIG");
+   configPath += "/config_percep/config.json";
+   configFile.open( configPath );
+   string config = "";
+   string setting;
+   while( configFile >> setting )
+       {
+          config += setting;
+       }
+   configFile.close();
+ 
+   mRoverConfig.Parse( config.c_str() );
+
```

Comment:
Can you fix the indentation here? Also, can you change the comment on the top to be formatted the same as the other sections are?